### DISCUSSION TITLE
voice played slowly when mulit output is used incluing bluetooth headset and blackhole.

### DISCUSSION THREAD

**Discussion by dttlgotv** [ExistentialAudio/BlackHole#605] [Q&A]
I create a multi output include a bluetooth and BlackHole, voice can played well. But When I open a microphone using this bluetooth headset , voice file can not be played well. It is played slowly.

I test it on audacity. Same issue can be seen. If bluetooth micphone can not be opened, multioutput works well.

This issue can be seen on all macosx os.

**Comment by devinroth** [comment]
This has been discussed a few times. In a nutshell it has to do with the sample rate of the mic.

Devin Roth

**Comment by dttlgotv** [comment]
any detail？thanks a lot



发自我的iPhone


------------------ Original ------------------
From: Devin Roth ***@***.***&gt;
Date: Fri,Jun 25,2021 10:16 PM
To: ExistentialAudio/BlackHole ***@***.***&gt;
Cc: dttlgotv ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [ExistentialAudio/BlackHole] voice played slowly when mulit output is used incluing bluetooth headset and blackhole. (#428)





 This has been discussed a few times. In a nutshell it has to do with the sample rate of the mic.
 
 Devin Roth
 On Jun 25, 2021, 2:15 AM -0700, dttlgotv ***@***.***&gt;, wrote:
 &gt; I create a multi output include a bluetooth and BlackHole, voice can played well. But When I open a microphone using this bluetooth headset , voice file can not be played well. It is played slowly.
 &gt; I test it on audacity. Same issue can be seen. If bluetooth micphone can not be opened, multioutput works well.
 &gt; This issue can be seen on all macosx os.
 &gt; —
 &gt; You are receiving this because you are subscribed to this thread.
 &gt; Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by dttlgotv** [comment]
> This has been discussed a few times. In a nutshell it has to do with the sample rate of the mic. Devin Roth
> […](#)
> On Jun 25, 2021, 2:15 AM -0700, dttlgotv ***@***.***>, wrote: I create a multi output include a bluetooth and BlackHole, voice can played well. But When I open a microphone using this bluetooth headset , voice file can not be played well. It is played slowly. I test it on audacity. Same issue can be seen. If bluetooth micphone can not be opened, multioutput works well. This issue can be seen on all macosx os. — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub, or unsubscribe.

I am sure that I use same sample rate in multi output like micphone.
